experiment/simulation/simulation.html

```diff
@@ -0,0 +1,319 @@
+					<div id="one-one">
+						<div id="path1">
+							<svg width="256" height="256" viewBox="0 0 256 256">
+								<path fill="none" stroke="currentColor" stroke-width="1"
+									d="M 13.544914,0 78.032901,0.75866 c 0,0 46.153515,-1.85739 43.310645,45.26492 v 7,0 l 52.37123,-0.48233" />
+							</svg>
+							<div class="square blue">1</div>
+						</div>
+						<div class="square black xor">XOR
+						</div>
+						<div id="path2">
+							<svg width="256" height="256" viewBox="0 0 256 256">
+								<path fill="none" stroke="currentColor" stroke-width="1"
+									d="M 13.544914,0 48,0 48,110 78.032901,110.75866 c 0,0 46.153515,-1.85739 43.310645,45.26492 v 7,0 l 52.37123,-0.48233" />
+							</svg>
+							<div class="square orange">1</div>
+						</div>
+						<div id="path3">
+							<svg width="256" height="256" viewBox="0 0 256 256">
+								<path fill="none" stroke="currentColor" stroke-width="1"
+									d="M 13.544914,100 38,100 38,210 78.032901,210.75866 c 0,0 46.153515,1.85739 43.310645,-45.26492" />
+							</svg>
+							<div class="square red">1</div>
+						</div>
+						<div class="square black and">AND
+						</div>
+						<div id="path4">
+							<svg width="256" height="256" viewBox="0 0 256 256">
+								<path fill="none" stroke="currentColor" stroke-width="1"
+									d="M 13.544914,100 78.032901,100 c 0,0 46.153515,1.85739 43.310645,-45.26492" />
+							</svg>
+							<div class="square green">1</div>
+						</div>
+					</div>
+					<div id="one-zero">
+						<div id="path1">
+							<svg width="256" height="256" viewBox="0 0 256 256">
+								<path fill="none" stroke="currentColor" stroke-width="1"
+									d="M 13.544914,0 78.032901,0.75866 c 0,0 46.153515,-1.85739 43.310645,45.26492 v 7,0 l 52.37123,-0.48233" />
+							</svg>
+							<div class="square blue">0</div>
+						</div>
+						<div class="square black xor">XOR
+						</div>
+						<div id="path2">
+							<svg width="256" height="256" viewBox="0 0 256 256">
+								<path fill="none" stroke="currentColor" stroke-width="1"
+									d="M 13.544914,0 48,0 48,110 78.032901,110.75866 c 0,0 46.153515,-1.85739 43.310645,45.26492 v 7,0 l 52.37123,-0.48233" />
+							</svg>
+							<div class="square orange">0</div>
+						</div>
+						<div id="path3">
+							<svg width="256" height="256" viewBox="0 0 256 256">
+								<path fill="none" stroke="currentColor" stroke-width="1"
+									d="M 13.544914,100 38,100 38,210 78.032901,210.75866 c 0,0 46.153515,1.85739 43.310645,-45.26492" />
+							</svg>
+							<div class="square red">1</div>
+						</div>
+						<div class="square black and">AND
+						</div>
+						<div id="path4">
+							<svg width="256" height="256" viewBox="0 0 256 256">
+								<path fill="none" stroke="currentColor" stroke-width="1"
+									d="M 13.544914,100 78.032901,100 c 0,0 46.153515,1.85739 43.310645,-45.26492" />
+							</svg>
+							<div class="square green">1</div>
+						</div>
+						<!-- <div id="pp">
```

Comment:
Please remove dead code.

Reply:
Removed all the dead code from all the files.